zk/src/archive/web/js/zk/au.js

```diff
@@ -1642,10 +1642,17 @@ zAu.cmd1 = /*prototype*/ {
 	addChd: function (wgt) {
 		if (wgt) {
 			var fn = function (child) {
-				var act = _beforeAction(child, 'show');
+				var act = _beforeAction(child, 'show'),
```

Comment:
should consider a change in addChd function flow, other component also could do fireSizedLater, not only Listbox

Reply:
ok